app/src/main/java/com/android/streetworkapp/ui/profile/AddFriend.kt

```diff
@@ -88,7 +96,12 @@ fun AddFriendScreen(
         username = user.username,
         onAccept = {
           userViewModel.addFriend(uid, receivedRequestUid)
-          Toast.makeText(context, "Friend added.", Toast.LENGTH_SHORT).show()
+          // Toast.makeText(context, "Friend added.", Toast.LENGTH_SHORT).show()
+          if (host != null && navigationActions != null) {
```

Comment:
no need check if navigationsActions is non-null, it's always the case when passed from MainActivity

Reply:
This as been updated, thank you!